docs/F-Refinements/portability.md

```diff
@@ -182,35 +184,35 @@ lint.c(11) : warning C4244: '=' : conversion from 'int' to 'char', possible loss
 
 You can find documentation on lint code checkers at
 
-* [Lint](http://www.unix.com/man-page/FreeBSD/1/lint)
-* [LCLint/Splint](http://lclint.cs.virginia.edu/)
+- [`Lint`](http://www.unix.com/man-page/FreeBSD/1/lint/ "Linux and UNIX Man Pages")
+- [`LCLint/Splint`](http://lclint.cs.virginia.edu/ "Offical page for the Splint tool")
```

Comment:
There's a typo in the title of this link, "Offical" instead of "Official"

Reply:
Hi there, I just fixed it. Please review it again. Thank you